lib/websocket.js

```diff
@@ -3,17 +3,31 @@
 var levels = require('./levels');
 var times = require('./times');
 var calcData = require('./data/calcdelta');
-
+var ObjectID = require('mongodb').ObjectID;
+       
 function init (env, ctx, server) {
 
   function websocket ( ) {
     return websocket;
   }
 
+  var log_yellow = '\x1B[33m';
+  var log_reset = '\x1B[0m';
+  var LOG_IO = log_yellow + 'IO: ' + log_reset;
```

Comment:
thinking about this a little more, instead of IO we should use WS, and maybe green, since yellow is usually a warning